src/arc.ts

```diff
@@ -80,6 +81,18 @@ async function _getCurrentAccountFromProvider(web3?: any): Promise<string> {
   return accounts[0] ? accounts[0].toLowerCase() : null;
 }
 
+/**
+ * Return the most recently synced block from web3
+ */
+export async function getCurrentBlock(web3?: any): Promise<any> {
```

Comment:
should this function exported ? why ?

Reply:
oh i was using it externally. now im not, though i could easily imagine it being used externally in the future. for now i can remove

Reply:
I suspect `getCurrentBlock` should be contained in subgraphHelpers, and not be exported.  Seems like this may be the only place it is used.

Reply:
of course changing the means I now have to export getWeb3, so not really any benefit here. to me putting getCurrentBlock in the same place as the other web3 functions makes more sense to me

Reply:
makes more sense to me to export getWeb3.

Reply:
but I'm not attached to it